exporter/googlecloudpubsubexporter/exporter.go

```diff
@@ -67,7 +65,7 @@ func (ex *pubsubExporter) start(ctx context.Context, _ component.Host) error {
 	ctx, ex.cancel = context.WithCancel(ctx)
 
 	if ex.client == nil {
-		client, err := newPublisherClient(ctx, ex.config, ex.userAgent)
+		client, err := ex.makeClient(ctx, ex.config, ex.userAgent)
```

Comment:
I don't find where the makeClient function gets initialized in the exporter (I do see it in the tests).

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/aaece40595d225d44667a1b95107b484e734a5a6/exporter/googlecloudpubsubexporter/factory.go#L56 ? 
Anyway , I see I'll have to run go tidy